tworaven_apps/eventdata_queries/models.py

```diff
@@ -40,6 +49,11 @@ class EventDataSavedQuery(TimeStampedModel):
     modified = models.DateTimeField(auto_now_add=True)
     saved_to_dataverse = models.BooleanField(default=False)
     dataverse_url = models.URLField(blank=True)
+    dataset = models.TextField(blank=True)
```

Comment:
Remove `saved_to_dataverse` and `dataverse_url`
- Make any related updates where these attributes are used